src/CorpusLoaders.jl

```diff
@@ -13,24 +13,13 @@ export load
 
 export WikiCorpus, SemCor, Senseval3, CoNLL, IMDB, Twitter, StanfordSentimentTreebank, WikiGold, CoNLL2000
 
-function __init__()
-    include(joinpath(@__DIR__, "WikiCorpus_DataDeps.jl"))
-    include(joinpath(@__DIR__, "SemCor_DataDeps.jl"))
-    include(joinpath(@__DIR__, "SemEval2007Task7_DataDeps.jl"))
-    include(joinpath(@__DIR__, "Senseval3_DataDeps.jl"))
-    include(joinpath(@__DIR__, "CoNLL_DataDeps.jl"))
-    include(joinpath(@__DIR__, "IMDB_DataDeps.jl"))
-    include(joinpath(@__DIR__, "Twitter_DataDeps.jl"))
-    include(joinpath(@__DIR__, "StanfordSentimentTreebank_DataDeps.jl"))
-    include(joinpath(@__DIR__, "WikiGold_DataDeps.jl"))
-    include(joinpath(@__DIR__, "CoNLL2000_DataDeps.jl"))
-end
 
 include("types.jl")
 include("apply_subparsers.jl")
 include("WikiCorpus.jl")
 include("SemCor.jl")
 include("SemEval2007Task7.jl")
+include("SemEval2007Task7_DataDeps.jl")
 include("Senseval3.jl")
 include("CoNLL.jl")
```

Comment:
Why not declare it inside the file?

Reply:
Ok, that can also be done. I left the empty file as it is, to accommodate any changes or missing content. But as you suggest, I will combine both the files into `SemEval2007Task7.jl` file